nodebuilder/tests/blob_test.go

```diff
@@ -123,6 +123,37 @@ func TestBlobModule(t *testing.T) {
 				require.ErrorContains(t, err, blob.ErrBlobNotFound.Error())
 			},
 		},
+		{
+			name: "Submit equal blobs",
+			doFn: func(t *testing.T) {
+				appBlob, err := blobtest.GenerateV0Blobs([]int{8, 4}, true)
+				require.NoError(t, err)
+				b, err := blob.NewBlob(
+					appBlob[0].ShareVersion,
+					append([]byte{appBlob[0].NamespaceVersion}, appBlob[0].NamespaceID...),
+					appBlob[0].Data,
+				)
+				require.NoError(t, err)
+				height, err := fullClient.Blob.Submit(ctx, []*blob.Blob{b, b}, nil)
+				require.NoError(t, err)
+				_, err = fullClient.Header.WaitForHeight(ctx, height)
+				require.NoError(t, err)
```

Comment:
Doesn't blob.Get do it automatically?

Reply:
It does. Not the `blob.Get`, the `header.Store`.

Reply:
I mean why do we need to wait here if Get waits in the same way?

Reply:
because we can get this error:
`header: given height is from the future:`

Reply:
Ahh, I see, thank you. I forgot it works through our service